manifest.konnector

```diff
@@ -1,33 +1,29 @@
 {
-  "version": "3.0.0",
+  "version": "0.1.0",
   "name": "Trainline",
   "type": "node",
```

Comment:
We have to change that type to `konnector` when our backend will be able to handle the `language: node` instead.